product_profile/__openerp__.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
```

Comment:
thanks, header is now ok with guidelines 
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#differences-with-odoo-guidelines
and main python standard
https://www.python.org/dev/peps/pep-0263/